apiml-static-reg.yaml.template

```diff
@@ -12,7 +12,7 @@ services:
     apiInfo:
       - apiId: org.zowe.zss
         gatewayUrl: api/v1
-        # swaggerUrl: TODO
+        swaggerUrl: https://${ZOWE_EXPLORER_HOST}:${ZOWE_ZLUX_SERVER_HTTPS_PORT:-8544}/api-docs/agent
```

Comment:
/server/agent/swagger as per comments in https://github.com/zowe/zlux-server-framework/pull/279

Reply:
`/server/agent` endpoints seems protected via rbac setting
![image](https://user-images.githubusercontent.com/4791635/117404488-02518d00-aed8-11eb-91f5-3afa6be7fb7d.png)